src/engraving/libmscore/ambitus.cpp

```diff
@@ -679,6 +679,16 @@ Ambitus::Ranges Ambitus::estimateRanges() const
     return result;
 }
 
+void Ambitus::remove(EngravingItem* e)
+{
+    if (e->type() == ElementType::ACCIDENTAL) {
+        //! NOTE Do nothing (removing _topAccid or _bottomAccid)
```

Comment:
Why doing nothing? Via Inspector the Ambitus can get changed to a non-accidental note, why not via deleting the accidental here?

Reply:
Because they are deleted in
```
Ambitus::~Ambitus()
{
    delete _topAccid;
    delete _bottomAccid;
}
```

They hide somehow differently (I did not investigate in detail)

Reply:
But, using you patch, they don't disapear, so don't delete at all, unless you delete the entire Ambitus (only that would call the destructor)

Reply:
I had no plans to change the implementation and logic of `Ambitus`.
I just avoided triggering the assertion, that is, I didn’t make worse than it was, I did it a little better.
Do you think it is necessary to reconsider the implementation of `Ambitus`?

Reply:
Well, doing nothing here is certainly better than a crash